src/App.js

```diff
@@ -18,28 +20,43 @@ const App = () => {
     e.preventDefault();
     await fetch(base + 'addPerson', {
       method: 'POST',
-      headers: {'Content-Type': 'application/json'},
+      headers: {'Content-Type': appJson},
       body: JSON.stringify({
         email: emailRef.current.value,
-        password: passwordRef.current.value
+        password: passwordRef.current.value,
+        admin: adminState
       })
     });
     // TODO: need to show errors if they happen
     await loadPeople(setPeople);
   };
 
+  const changeAdmin = id => async e => {
+    e.preventDefault();
```

Comment:
You don't want to prevent the default here. The default is that the checkbox gets checked. That's why there's a delay when you click it.

Reply:
took it out here and at delete user..